plugin/ochttp/trace.go

```diff
@@ -40,7 +40,7 @@ const (
 )
 
 type traceTransport struct {
-	base   http.RoundTripper
+	*Transport
```

Comment:
Can we instead have a Sampler field? Circular dependencies between traceTransport and Transport and statsTransport and Transport are not ideal.